devbook.xsl

```diff
@@ -542,16 +554,16 @@
                 </button>
               </div>
               <div class="collapse navbar-collapse" id="gw-toolbar">
-              <div class="input-group">
-                <input type="search" name="search" placeholder="Search" title="Search Gentoo Developer Manual [f]" accesskey="f" id="searchInput" class="form-control" onclick="fetchDocuments()"/>
-                <div class="input-group-btn">
-                  <input type="submit" name="fulltext" value="Search" title="Search the pages for this text" id="mw-searchButton" class="searchButton btn btn-default" onclick="search()"/>
+                <div class="input-group">
+                  <input type="search" name="search" placeholder="Search" title="Search Gentoo Developer Manual [f]" accesskey="f" id="searchInput" class="form-control" onclick="fetchDocuments()"/>
```

Comment:
Again, it's an external resource. I cannot predict in devbook.xsl if it will be available for the user, and suppress the search elements if it is not.